services/explorer/src/api.rs

```diff
@@ -512,11 +511,11 @@ impl ExplorerApi {
             })
     }
 
-    fn transaction_status(
-        schema: Schema<&dyn Snapshot>,
+    async fn transaction_status(
```

Comment:
Adding to previous comments: IMO, it makes little sense to make a function async just because you can (especially if the function's private). Doing this would be more transparent:

```rust
.endpoint("v1/call_status/transaction", |state, query| async {
    Self::transaction_status(state.data().for_core(), query)
})
```

Reply:
But with async functions I can keep `wire_rest` boilerplate-less without any side effects.